web_console/web_terminal/troubleshooting-web-terminal.adoc

```diff
@@ -9,6 +9,44 @@ toc::[]
 [id="troubleshooting-web-terminal-network-policies"]
 == Web terminal and network policies
 
-The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state.
+The web terminal might fail to launch if the cluster has network policies configured. To initialize a web terminal instance, the {web-terminal-op} must communicate with the web terminal's pod to verify it is running, and the {product-title} web console needs to send information to automatically log in to the cluster within the terminal. If either step fails, the web terminal fails to initialize and the terminal panel appears to be in a loading state until a context deadline exceeded error occurs.
```

Comment:
Is there a way to reformat "context deadline exceed" to let readers know it is a type of error? Maybe something like "...until a "context deadline exceeded" error occurs" or "...until a `context deadline exceeded` error occurs".

Reply:
Sounds good, I've updated to use the latter:
![image](https://github.com/user-attachments/assets/264d19b3-e240-4ffa-8179-ae139a594c78)